packages/worker/src/worker-options.ts

```diff
@@ -487,9 +487,10 @@ export interface WorkerOptions {
    * From running basic stress tests we've observed 2/3 reduction in memory usage and 1/3 to 1/2 in CPU usage with this
    * feature turned on.
    *
-   * Note that we plan to turn this option on by default starting with 1.9.0.
+   * WARNING: There is currently no known reason to turn this off. The non-`reuseV8Context` execution model will be
+   * completely removed in 1.10.0. Please update your worker options to use `reuseV8Context: true` as soon as possible.
```

Comment:
Wait, doesn't this imply they want to _remove_ it from their worker options now?